src/StoreKeeper/WooCommerce/B2C/Tools/Attributes.php

```diff
@@ -294,7 +294,8 @@ public static function updateAttributeAndOptionFromContentVar($content_var)
                 $attribute_option_id,
                 $attribute_option_slug,
                 $attribute_option_name,
-                $attribute_option_image
+                $attribute_option_image,
+                null,
```

Comment:
```suggestion
                0,
```

Reply:
Changed to variable with `0` as fallback.